Tests/Tracking/APITest.swift

```diff
@@ -1,4 +1,5 @@
 import CioTracking // do not use `@testable` so we can test functions are made public and not `internal`.
+@testable import Common
```

Comment:
```suggestion
```

The `APITest.swift` files are unique in that they are the only test classes in the code base that should not
* use `@testable` on any of the imports. 
* import a module that we don't expect customers to use themselves. 

These rules are put into place to prevent false positive tests. 

`APITest`s are designed to test the public API that customers would use. 

I tried adding some comments to these files with these rules but we can probably improve them for future devs besides yourself. Thanks for bringing this to our attention on something we can improve.